processor/pom.xml

```diff
@@ -57,7 +57,12 @@
             <artifactId>mapstruct</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
```

Comment:
I'm just starting to play around with #691, which would then test 1.7 and 1.8 language level... I don't remember why we're not testing language level 1.6 with JDT in the integration-tests, though.